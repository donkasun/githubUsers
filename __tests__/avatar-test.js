'use strict';

import renderer from 'react-test-renderer';
import Avatar from '../app/components/avatar';

it('renders correctly', () => {
  const tree = renderer.create(<Avatar />).toJSON();
  expect(tree).toMatchSnapshot();
});
