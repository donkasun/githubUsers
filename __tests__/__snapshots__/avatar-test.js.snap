// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  defaultSource={
    Object {
      "testUri": "../../../app/assets/images/user.png",
    }
  }
  source={
    Object {
      "uri": undefined,
    }
  }
  style={
    Object {
      "height": 30,
      "width": 30,
    }
  }
/>
`;
